pkg/updateengine/status.go

```diff
@@ -18,6 +18,18 @@ import (
 	"fmt"
 )
 
+// The possible update statuses returned from the update engine
+const (
```

Comment:
I see these are listed (only in comments) in pkg/constants/constants.go as well, though I kinda prefer them here. Pretty sure they correspond to current operation values so maybe specify that as well.